Quintor/src/main/java/com/example/quintor/TranscriptFile.java

```diff
@@ -0,0 +1,115 @@
+package com.example.quintor;
+
+import java.util.ArrayList;
+
+public class TranscriptFile {
```

Comment:
Is het niet duidelijk als we bankStatement gebruiken in plaats van transcriptFile?